common/tier4_planning_rviz_plugin/include/mission_checkpoint/mission_checkpoint.hpp

```diff
@@ -67,21 +67,19 @@ class MissionCheckpointTool : public rviz_default_plugins::tools::PoseTool
 
 public:
   MissionCheckpointTool();
-  virtual ~MissionCheckpointTool() {}
-  virtual void onInitialize();
+  void onInitialize() override;
 
 protected:
-  virtual void onPoseSet(double x, double y, double theta);
+  void onPoseSet(double x, double y, double theta) override;
 
 private Q_SLOTS:
   void updateTopic();
 
-private:
+private:  // NOLINT for Qt
```

Comment:
Where is Qt?

Reply:
Actually  I want to write with `private Q_SLOTS`, but clang-tidy still generates warning.

Reply:
Hmm? What is the warning?

Reply:
Now I have to write with `private:`

Reply:
The `readability-redundant-access-specifiers` occurs

Reply:
Ah, I see. It's for this block.
![image](https://user-images.githubusercontent.com/31987104/185592554-604d3806-5851-4fed-a0ef-ca0d4940aa6f.png)